### DISCUSSION TITLE
Come Build/Play a Game with Me Next Week

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#15] [Comments]
# Come Build/Play a Game with Me Next Week

Announcing my next <Code><Br> Episode

https://www.raymondcamden.com/2024/02/13/come-buildplay-a-game-with-me-next-week

**Comment by robertz** [comment]
Bringing back the Idle Ships game?

**Reply by cfjedimaster** [reply]
I'm going to share it, but for this I'm building something very simple from scratch.
